azext_connect/commands.py

```diff
@@ -5,10 +5,12 @@ def load_command_table(self, _):
 
     with self.command_group('cupertino webapp') as g:
         g.custom_command('bind', 'bind_webapp')
+        g.custom_command('validate', 'validate_general')
```

Comment:
the validate command which only need connection name is not bond to any service right? So put in command_group "cupertino" ?

Reply:
ah, making great sense. let me do that.